Client/Frontend/Home/TopSitesPanel.swift

```diff
@@ -260,7 +260,14 @@ extension TopSitesPanel: UICollectionViewDelegate {
         if let site = dataSource[indexPath.item] {
             // We're gonna call Top Sites bookmarks for now.
             let visitType = VisitType.Bookmark
-            let destination = NSURL(string: site.url)?.domainURL() ?? NSURL(string: "about:blank")!
+
+            let destination: NSURL
+            if site is SuggestedSite {
```

Comment:
Sadly extension functions/vars can't be overriden yet :( I'll add it to the class and override.